src/NServiceBus.Gateway.AcceptanceTests/GatewayEndpoint.cs

```diff
@@ -13,7 +13,8 @@ public override Task<EndpointConfiguration> GetConfiguration(RunDescriptor runDe
             {
                 var endpointName = endpointCustomizationConfiguration.CustomEndpointName ?? configuration.GetSettings().EndpointName();
 
-                var deduplicationConfiguration = GatewayTestSuiteConstraints.Current.ConfigureDeduplicationStorage(
+                var constraints = new GatewayTestSuiteConstraints();
```

Comment:
would it make sense to just snapshot the instance here?

```suggestion
                var constraints = GatewayTestSuiteConstraints.Current;
```
I'm not sure about the usage of the current static property, but IIRC in core, this can also be used to replace default settings when needed on a specific test? not sure I remember the details correctly though.